cellpack/bin/simularium_converter.py

```diff
@@ -237,7 +243,7 @@ def unpack_curve(self, data, time_step_index, ingredient_name, index, agent_id):
             self.max_fiber_length = len(data[curve])
 
     def unpack_positions(
-        self, data, time_step_index, ingredient_name, index, agent_id, comp_id=0
+        self, data, time_step_index, ingredient_name, index, agent_id, comp_id=0, left_hand=False
```

Comment:
is this only for blender OBJs? does it work with meshes exported from other 3D software?

Reply:
sorry I forgot to remove it. The problem was indeed from the blender export and should not be the case in other software. 
The scaling is an issues, and you should be able to pass the scaling factor to simularium. so that simularium does the scaling not the converter. Assuming you want both geom and PDBs, for instance a mesh for the compartment and PDBs for the ingredient, you want to be able to have the PDb and the geom at the same scale.

Reply:
otherwise could the scaling factor be an argument to the convert command line ?

Reply:
I'm confused by the scaling. The OBJs I had before were working with the .1 scale factor. Now with a scale factor of 1 the pdbs are way to small, and the OBJs work, but the whole scene is also way too large for the simularium viewer (the objects are disappearing in the back) did you scale up these OBJ files?

Reply:
shouldn't this be internally consistent in cellPACK?

Reply:
I scaled them temporarly by 0.1 to make sure simularium was not scaling them, and then revert back to default scaling. So the problem here is simularium not being aware of the scaling factor used during the conversion. The rotation issues is a deeper problem on how it was saved. If it was saved left-hand/righthand and if it was saved using transformation.py. If you want to use scipy, change the saving code that generate the quaternion so its consistent. transformation.py and scipy gave different results during the euler<->quaternion convertion for the same axis,angle rotation. because the quaternion order is different x,y,z,w in scipy and likely w,x,y,z in transformation.py.